deploy.sh

```diff
@@ -78,11 +78,20 @@ MVN_OPTIONS+=(
   "--batch-mode"
 )
 
+echo "[INFO] mvn clean compile"
+mvn clean compile ${MVN_OPTIONS[@]}
+
+echo "[INFO] mvn dependency resolve"
+mvn dependency:resolve dependency:resolve-plugins dependency:go-offline -DmanualInclude=org.apache.maven:maven-archiver:pom:2.5 ${MVN_OPTIONS[@]}
```

Comment:
I'm not convinced that this dependency is causing the issue, should we remove this and try if the deploy works or not first?